### ISSUE TITLE
black upscale

### ISSUE THREAD

**Issue by Marvin1500** [upscayl/upscayl#362]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#-faq).
- [X] I have checked that this issue isn't addressed in any other issue or pull request.

### Describe the Bug

hello,
first, i'm french so, sorry for my english... :)
i didn't use upscayl since a few weeks, and this morning when i started it there was an update (V 2.5.5)

now, when i upscayl a picture i have a black picture...

someone can help me?

thx ;)

### To Reproduce

.

### Expected Behavior

.

### Version/Commit Hash

2.5.5

### OS Specifics

windows

### Screenshots

<a href="https://zupimages.net/viewer.php?id=23/24/3es4.png"><img src="https://zupimages.net/up/23/24/3es4.png" alt="" /></a>

**Comment by NayamAmarshe** [comment]
Please check the wiki. https://github.com/upscayl/upscayl/wiki

**Comment by Marvin1500** [comment]
yes but i don't understand what is my problem... here is the logs:

⚙️ Getting saveImageAs from localStorage jpg
⚙️ Getting model from localStorage {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Getting gpuId from localStorage 1
🔀 Setting model to 
🔄 Resetting image paths
🔀 Setting model to realesrgan-x4plus
⚙️ Getting saveImageAs from localStorage jpg
⚙️ Getting model from localStorage {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Getting gpuId from localStorage 1
⚙️ Getting rememberOutputFolder from localStorage false
⚙️ Getting saveImageAs from localStorage jpg
⚙️ Getting model from localStorage {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Getting gpuId from localStorage 1
🔀 Setting model to 
🔀 Setting model to realesrgan-x4plus
🔄 Resetting image paths

🐞 BACKEND REPORTED:  📢 Upscayl Command:  -i,C:\Users\lebro\OneDrive\Images\Marvin Dgn\à upscale\1\marvin_dixon_computer_mouse_1d09def3-5379-4576-a03a-f8ec88c4989a.png,-o,C:\Users\lebro\OneDrive\Images\Marvin Dgn\upscaled\marvin_dixon_computer_mouse_1d09def3-5379-4576-a03a-f8ec88c4989a_upscayl_4x_realesrgan-x4plus.jpg,-s,4,-m,C:\Program Files\Upscayl\resources\models,-n,realesrgan-x4plus,-g,0,1,2,-f,jpg
🐞 BACKEND REPORTED:  image upscayl:  [0 NVIDIA GeForce GTX 950M]  queueC=0[16]  queueG=0[16]  queueT=1[1]
[0 NVIDIA GeForce GTX 950M]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 NVIDIA GeForce GTX 950M]  fp16-p/s/a=1/1/0  int8-p/s/a=1/1/1
[0 NVIDIA GeForce GTX 950M]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🚧 UPSCAYL_PROGRESS:  [0 NVIDIA GeForce GTX 950M]  queueC=0[16]  queueG=0[16]  queueT=1[1]
[0 NVIDIA GeForce GTX 950M]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[0 NVIDIA GeForce GTX 950M]  fp16-p/s/a=1/1/0  int8-p/s/a=1/1/1
[0 NVIDIA GeForce GTX 950M]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

🐞 BACKEND REPORTED:  image upscayl:  [1 Intel(R) HD Graphics 620]  queueC=0[1]  queueG=0[1]  queueT=0[1]
[1 Intel(R) HD Graphics 620]  bugsbn1=0  bugbilz=0  bugcopc=0  bugihfa=0
[1 Intel(R) HD Graphics 620]  fp16-p/s/a=1/1/0  int8-p/s/a=1/1/0
[1 Intel(R) HD Graphics 620]  subgroup=32  basic=1  vote=1  ballot=1  shuffle=1

**Comment by aaronliu0130** [comment]
Have you tried [general fixes](https://github.com/upscayl/upscayl/wiki/Troubleshooting#general-fixes)?

**Comment by Marvin1500** [comment]
yes i did that but no difference

**Comment by aaronliu0130** [comment]
Do you have the right redistributables installed?

**Comment by Marvin1500** [comment]
Sorry, I don't have a great deal of knowledge on the subject. i don't know what it is. how can i know if i have the rights ?

**Comment by aaronliu0130** [comment]
Just search "redistributable" under installed programs and send a screenshot

**Comment by Marvin1500** [comment]
i'm very sorry for my ignorance, but where exactly do i have to tip my research?

**Comment by aaronliu0130** [comment]
Just search in your task bar "installed apps" and opening it and then search "redistributable" in that

**Comment by Marvin1500** [comment]
ok, i think it's that : 
<a href="https://zupimages.net/viewer.php?id=23/25/74fs.png"><img src="https://zupimages.net/up/23/25/74fs.png" alt="" /></a>

**Comment by aaronliu0130** [comment]
Try downloading the latest redist from https://aka.ms/vs/17/release/vc_redist.x64.exe .

**Comment by Marvin1500** [comment]
no this is not better with that... :(

**Comment by Marvin1500** [comment]
does anyone have any other ideas?

**Comment by NayamAmarshe** [comment]
Some hardware is simply incompatible. GTX950M is an old card. It's possible it's incompatible with Real-ESRGAN.

**Comment by Marvin1500** [comment]
yes , this is the first thing i tried but it doesn't work...

**Comment by NayamAmarshe** [comment]
Very strange. I'm not sure how to fix it either. All I can think of is trying Upscayl on another computer or reinstalling graphics drivers with NVCleanInstall.

**Comment by Marvin1500** [comment]
it work when i choose 0 in GPU ID. i don't understand because i am with Nvidia Geforce...

**Comment by NayamAmarshe** [comment]
> it work when i choose 0 in GPU ID. i don't understand because i am with Nvidia Geforce... and the wiki say 1 is for Nvidia .0 for AMD radeon. is there an error in the wiki?
> anyway it works... thx for your time ;)
> 

No, the wiki is just giving an example, you have to see your own ID in the logs.

I'm glad it's working for you now :)

**Comment by Ig-abdullah01** [comment]
same problem can anyone tell solution for this issue <?

**Comment by Tomac12** [comment]
I also had the problem, I then found out that it was due to the GUI. I set this to “BLACK”. Since then it works without problems.

**Comment by zalmdain-ship-it** [comment]
Cause of the problem The device has two cards 
Intel HD Graphics
NVIDIA GeForce
Effective Solutions : 
1) Update graphics card definitions / After installation, restart the device.
2) Always run Upscayl with NVIDIA card only Open Nvidia: **NVIDIA Control Panel → Manage 3D Settings → Program Settings → Upscayl.exe → High-performance NVIDIA processor**
3) Install / reinstall Vulkan Runtime - Download the latest version suitable for Windows and then restart your device.
4) Change Upscayl settings within the app / Try the following: From Settings choose **GPU = 1** (NVIDIA card number) or try **GPU = 0**
_If all these solutions fail try to choose the **model light** in the main interface of the program this will temporarily solve the problem_
